templates/taxi/car_detail.html

```diff
@@ -3,13 +3,21 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
+    <a href="{% url "taxi:car-delete" pk=car.id %}" class="btn btn-danger link-to-page">
       Delete
     </a>
 
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
+    <a href="{% url "taxi:car-update" pk=car.id %}" class="btn btn-secondary link-to-page">
       Update
     </a>
+    {% if is_driver %}
+    <a href="{% url "taxi:remove_from_car" car.id %}">Delete me from this car
```

Comment:
The URL tag for 'remove_from_car' is missing the `pk` keyword for the parameter. It should be `{% url "taxi:remove_from_car" pk=car.id %}`.